pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -158,14 +158,18 @@ stdenv.mkDerivation rec {
     "-Dpulse=disabled" # TODO check if we can keep this enabled
     "-Dv4l2-gudev=disabled" # Linux-only
     "-Dv4l2=disabled" # Linux-only
-    "-Dximagesrc=disabled" # Linux-only
   ] ++ lib.optionals (!raspiCameraSupport) [
     "-Drpicamsrc=disabled"
   ];
 
   postPatch = ''
     patchShebangs \
       scripts/extract-release-date-from-doap-file.py
+  ''
+  # gstreamer-gl-* detection is broken due to an upstream meson issue
```

Comment:
Can you link the upstream meson issue in the comment here?

Reply:
I don't know if there is some. There is a code comment https://gitlab.freedesktop.org/gstreamer/gst-plugins-good/-/blob/discontinued-for-monorepo/meson.build#L321-324 which was added in https://gitlab.freedesktop.org/gstreamer/gst-plugins-good/-/merge_requests/680

Reply:
Is this substituteInPlace even necessary? It seems to have been a while ago, but I just tried this PR without that substituteInPlace and it seems to have built fine (just saying gstreamer-gl was not found and moving on)

Reply:
(other than that question, this PR looks good now btw, thank you!)

Reply:
Just tried again and it seems to work without this now, too.